<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReSetProp</class>
 <widget class="QDialog" name="ReSetProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chỉnh sửa thông tin</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="tenLabel">
        <property name="text">
         <string>Tên (không dấu không cách)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="giaLabel">
        <property name="text">
         <string>Đơn giá (VNĐ)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="donViLabel">
        <property name="text">
         <string>Đơn vị tính</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="loaiLabel">
        <property name="text">
         <string>Loại mặt hàng</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="tenLine"/>
      </item>
      <item>
       <widget class="QLineEdit" name="giaLine"/>
      </item>
      <item>
       <widget class="QComboBox" name="donViTinhBox">
        <item>
         <property name="text">
          <string>Ly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Xien</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dia</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="loaiHangBox">
        <item>
         <property name="text">
          <string>Tra</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NuocNgot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CaPhe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DoAn</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReSetProp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReSetProp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
